src/types.ts

```diff
@@ -570,6 +570,27 @@ export interface VaultAccountsFilter {
     assetId?: string;
 }
 
+export interface PagedVaultAccountsRequestFilters {
+    namePrefix?: string;
+    nameSuffix?: string;
+    minAmountThreshold?: number;
+    assetId?: string;
+    orderBy?: "ASC" | "DESC";
+    limit?: number; // for default and max limit values please see: https://docs.fireblocks.com/api/swagger-ui/#/
+    before?: string;
```

Comment:
not sure what this means : before \ after what ? (I understand this propertiers need to match how it is defined in the BE Api, but maybe worth some explanation through comments ?)